zipline/testing/fixtures.py

```diff
@@ -316,6 +316,10 @@ class WithAssetFinder(WithDefaultDateBounds):
         ``END_DATE``.
     ASSET_FINDER_EQUITY_NAMES: iterable[str]
         The default names to use for the equities.
+    ASSET_FINDER_EQUITY_EXCHANGE : str
```

Comment:
Same question as below re: calling this `ASSET_FINDER_DEFAULT_EQUITY_EXCHANGE` or similar.

Reply:
it's not really default, it is the value that will be used in `make_equity_info` just like `ASSET_FINDER_EQUITY_NAMES` is the value used in the same function.